src/cascadia/TerminalApp/TitlebarControl.idl

```diff
@@ -7,7 +7,7 @@ namespace TerminalApp
     {
         TitlebarControl(UInt64 parentWindowHandle);
 
-        Windows.UI.Xaml.UIElement Content;
+        IInspectable Content;

```

Comment:
Why?

Reply:
Content = IInspectable is the contract for ContentPresenter (in theory you can put a string into the Content, which is 100% valid)